users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_sort_layer2.py

```diff
@@ -46,6 +44,22 @@ def sort_filters_by_center_freq(x):
     # move to log domain, not needed for center frequencies
     # f_resp = 20 * tf.math.log(f_resp) / tf.math.log(tf.constant(10.0))
```

Comment:
Since we do not only sort but return the actual frequency response, let's uncomment this.
```suggestion
    f_resp = 20 * tf.math.log(f_resp) / tf.math.log(tf.constant(10.0))
```